app/src/main/java/com/example/filemanager/AllFilesFragment.kt

```diff
@@ -65,8 +82,25 @@ class AllFilesFragment : Fragment(), ItemClickListener {
         fetchFiles()
     }
 
-    override fun onItemClickListener() {
-        Toast.makeText(context, "Item clicked", Toast.LENGTH_SHORT).show()
+    private fun setToolbarNavigation() {
+        binding.toolbar.setNavigationIcon(R.drawable.back_icon)
+        binding.toolbar.setNavigationOnClickListener {
+            handleBackPress()
+        }
+    }
+
+    private fun hideToolbarNavigation() {
+        binding.toolbar.navigationIcon = null
```

Comment:
the same question here, is it the recommended way to hide the back button from toolbar?

Reply:
I am not sure, but it's the best way I found